doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -370,9 +372,8 @@ specified by the **geometry** parameter:
   is the center of the circle.
 + In the case of `polygon` includes a list of elements, which are the vertices of the polygon.
 
-# REST API
 
-# Group NGSIv2
+# Group Root
```

Comment:
I have created several groups instead of having only a group. I think that improves the rendering but Root can be changed. please check my proposal.